HW03/Nastja_Kireeva/spec/Nastja_Kireeva_hw_03_t_04_spec.rb

```diff
@@ -0,0 +1,24 @@
+require './Nastja_Kireeva_hw_03_t_04'
+
+RSpec.describe 'Task4' do
+  describe '.task4' do
+    context 'when string is given' do
+      it 'returns the count of letters and digits in the string' do
```

Comment:
```suggestion
      it 'returns the count of letters and digits in the string', :aggregate_failures do
```
https://relishapp.com/rspec/rspec-core/docs/expectation-framework-integration/aggregating-failures